diff --git a/node_modules/react-native-upi-payment/android/build.gradle b/node_modules/react-native-upi-payment/android/build.gradle
index f2b29ee..43cd26b 100644
--- a/node_modules/react-native-upi-payment/android/build.gradle
+++ b/node_modules/react-native-upi-payment/android/build.gradle
@@ -4,10 +4,11 @@ apply plugin: 'com.android.library'
 
 buildscript {
     repositories {
-        jcenter()
+        google()
+        mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:8.0.0'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
